### PR TITLE
phpdoc fix

### PR DESCRIPTION
**Pull request by akravets12**

### CODE REVIEW

File: src/ElementFinderInterface.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function content(string $expression, bool $outerContent = false): StringC
      * ```
      *
      * @param string $expression
-     * @return $this
+     * @return ElementFinder
```
Comment (funivan) [inline src/ElementFinderInterface.php]: @akravets12 why do you change the return type to ElementFinder?

Comment (akravets12) [reply src/ElementFinderInterface.php]: okay, its better to return ElementFinderInterface
ElementFinder is immutable and new realization of remove method returns ElementFinderInterface
```     * @param string $expression
     * @return ElementFinder
     */
    final public function remove($expression): ElementFinderInterface
    {
        return $this->modify($expression, new RemoveElements());
    }
    /**
     * @param string $expression
     * @param ElementFinderModifierInterface $modifier
     * @return ElementFinderInterface
     */
    public function modify(string $expression, ElementFinderModifierInterface $modifier): ElementFinderInterface
    {
        $result = clone $this;
        $modifier->modify(
            $result->query($expression)
        );
        return $result;
    }```

Comment (funivan) [reply src/ElementFinderInterface.php]: Return type should be `$this` or `ElementFinderInterface`. Prefer `ElementFinderInterface`

**Comment by funivan** [comment]
Thanks @akravets12

**Pull request merged**